test/images/Makefile

```diff
@@ -12,19 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all image push clean
+REGISTRY ?= k8s-e2e-images
```

Comment:
maybe on gcr.io, so `gcr.io/k8s-e2e-images`?
guess I should actually create that.
fyi @kubernetes/sig-testing-misc